fiftyone/core/cli.py

```diff
@@ -4194,6 +4195,57 @@ def execute(parser, args):
             fop.delete_plugin(name)
 
 
+class LabsCommand(Command):
+    """Tools for working with FiftyOne Labs."""
+
+    @staticmethod
+    def setup(parser):
+        subparsers = parser.add_subparsers(title="available commands")
+        _register_command(subparsers, "install", LabsInstallCommand)
+
+    @staticmethod
+    def execute(parser, args):
+        parser.print_help()
+
+
+class LabsInstallCommand(Command):
+    """Install fiftyone labs features from the web.
+
+    You can install all labs features (plugins) from GitHub with the following::
```

Comment:
nit: can you use the same `Examples::` syntax that you see in other `Command` classes in this module for consistency?